src/panel/menu/AppMenu.jsx

```diff
@@ -0,0 +1,62 @@
+/* globals _ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import MenuItem from './MenuItem';
+import Telemetry from 'src/libs/telemetry';
+import { Divider } from 'src/components/ui';
+import { Heart, IconLightbulb, IconEdit, IconApps } from 'src/components/ui/icons';
+import { PINK_DARK, ACTION_BLUE_BASE } from 'src/libs/colors';
+
+const AppMenu = ({ close, openProducts }) => {
+  const navTo = (url, options) => {
+    close();
+    window.app.navigateTo(url, options);
+  };
+
+  return <div className="menu-items">
+    <MenuItem
```

Comment:
Should we add a text-underline decoration below an anchor with a `menu-item` class ?